ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/conditionalBlock.component.ts

```diff
@@ -24,39 +23,15 @@ import { CompositeDisposable } from '../../../compositeDisposable';
                         (valueChanged)="handleChange($event)">
     </ddp-group-block>`
 })
-export class ConditionalBlockComponent implements OnInit, OnDestroy {
+export class ConditionalBlockComponent {
     @Input() block: ConditionalBlock;
     @Input() readonly: boolean;
     @Input() validationRequested: boolean;
     @Input() studyGuid: string;
     @Input() activityGuid: string;
     @Output() valueChanged: EventEmitter<AnswerValue> = new EventEmitter();
-    private anchor: CompositeDisposable;
-
-    constructor(private submissionManager: SubmissionManager) { }
-
-    public ngOnInit(): void {
-        const sub = this.submissionManager.answerSubmissionResponse$.subscribe((response) => {
-            this.updateVisibility(response.blockVisibility);
-        });
-        this.anchor = new CompositeDisposable(sub);
-    }
-
-    public ngOnDestroy(): void {
-        this.anchor.removeAll();
-    }
 
     public handleChange(value: AnswerValue): void {
         this.valueChanged.emit(value);
     }
-
-    public updateVisibility(visibility: BlockVisibility[]): void {
-        visibility.forEach(shownBlockStatus => {
-            const matchingBlock = this.block.nestedGroupBlock
-                .nestedBlocks.find(nestedBlock => nestedBlock.id === shownBlockStatus.blockGuid);
-            if (matchingBlock) {
-                matchingBlock.shown = shownBlockStatus.shown;
-            }
-        });
-    }
```

Comment:
ConditionalBlocks supposedly exist to control hide/show. It does provide structure to set this up, but reality is that that server can tell any block to hide or show any time. So all this code here is not necessary.